lib/transit_data/glides_report/util.ex

```diff
@@ -20,16 +20,17 @@ defmodule TransitData.GlidesReport.Util do
     |> String.pad_leading(count, "0")
   end
 
-  @spec group_by_hour(Enumerable.t({stop_id :: String.t(), timestamp :: integer})) :: %{
-          (hour :: 0..23) => MapSet.t({stop_id :: String.t(), minute :: 0..59})
-        }
-  def group_by_hour(stop_times) do
-    stop_times
-    |> Enum.group_by(
-      fn {_stop_id, timestamp} -> unix_timestamp_to_local_hour(timestamp) end,
-      fn {stop_id, timestamp} -> {stop_id, unix_timestamp_to_local_minute(timestamp)} end
-    )
-    |> Map.new(fn {hour, stop_minutes} -> {hour, MapSet.new(stop_minutes)} end)
+  @doc """
+  Formats the ratio of two numbers as a percentage.
+  """
+  @spec format_percent(number, number, String.t()) :: String.t()
+  def format_percent(_numerator, denominator, zero_fallback) when denominator == 0 do
```

Comment:
Did you use a guard because you felt it was more readable with the variable name of `denominator`? If so feel free to ignore this suggestion
```suggestion
  def format_percent(_numerator, 0, zero_fallback) do
```

Reply:
It's because pattern matching implicitly uses `===` which distinguishes between float 0.0 and integer 0, while `==` does not.

```ex
iex> comp = fn
       a, a -> "terms match"
       a, b when a == b -> "terms are equal"
       _, _ -> "terms are not equal"
     end

iex> comp.(0, 0)
"terms match"

iex> comp.(0.0, 0.0)
"terms match"

iex> comp.(0, 0.0)
"terms are equal"

iex> comp.(0, 1)
"terms are not equal"
```

Maybe the difference isn't a big deal here, but I figured I'd go with the "correct" behavior in case this function gets used in more places in the future.

Reply:
I can add a comment explaining this ^ since it's not at all obvious